Src/DependencyCollector/Shared/Implementation/ProfilerSqlProcessingBase.cs

```diff
@@ -101,6 +119,24 @@ public object OnEndForTwoParameters(object context, object returnValue, object t
         }
 
         /// <summary>
+        /// On end async callback for methods with 2 parameter.
+        /// </summary>
+        public object OnEndAsyncForTwoParameters(object context, object returnValue, object thisObj, object parameter1)
+        {
+            this.OnEndAsync(returnValue, thisObj);
+            return returnValue;
+        }
+
+        /// <summary>
+        /// On end async callback for methods with 2 parameter.
```

Comment:
both this method and the one right above it has exact same comment but does something different. Hence I asked to modify the comment to reflect what the function is doing.